pkg/chart/ChartService.go

```diff
@@ -100,7 +100,7 @@ type ChartService interface {
 	FlaggerCanaryEnabled(values json.RawMessage) (bool, error)
 	GetCustomChartInBytes(chatRefId int) ([]byte, error)
 	GetRefChart(templateRequest TemplateRequest) (string, string, error, string, string)
-	ExtractVariablesAndResolveTemplate(scope resourceQualifiers.Scope, template string, templateType parsers.VariableTemplateType) (string, error)
+	ExtractVariablesAndResolveTemplate(scope resourceQualifiers.Scope, template string, templateType parsers.VariableTemplateType, isSuperAdmin bool) (string, error)
```

Comment:
we should handle this through `ctx`

Reply:
will handle it properly in manager layer since here I need flag which tell to mask sensitive value. I'll create a request object for it